### DISCUSSION TITLE
How to show grid view, or vertical scroll view by preview icons.

### DISCUSSION THREAD

**Discussion by MasayaYamada** [GeekAbdelouahed/flutter-reaction-button#29] [Q&A]
Hi there,

I'm using the flutter_reaction_button. 
I would like to show the preview icons as grid view, or scroll view.
But I don't have any idea to implement to show the preview icons as grid or scroll view.
Could you let me know if you have any solutions?

https://stackoverflow.com/questions/70149051/how-to-show-grid-view-on-preview-icon-using-flutter-reaction-button

V/r,

**Comment by armmer016** [comment]
any update on this? I think the preview icon of the `flutter_reaction_button` should be customizable. It should be scrollable or use `GridView`.
